apps/test/unit/signUpFlow/FinishTeacherAccountTest.tsx

```diff
@@ -607,42 +628,11 @@ describe('FinishTeacherAccount', () => {
       const fetchCall = fetchStub.getCall(1);
       expect(fetchCall.args[0]).toEqual('/users');
       expect(fetchCall.args[1]?.body).toEqual(
-        JSON.stringify(finishSignUpParams)
+        JSON.stringify(FINISH_SIGN_UP_PARAMS)
       );
 
       // Verify the user is redirected to the finish sign up page
       expect(navigateToHrefMock).toHaveBeenCalledWith(userReturnToUrl);
     });
   });
-
-  it('requires educator role', async () => {
```

Comment:
Moved this test higher in the file so the tests are ordered by how they appear on the page (with full submission requests at the bottom).